app/models/runtime/helpers/service_credential_binding_mixin.rb

```diff
@@ -8,8 +8,8 @@ def operation_in_progress?
       !!last_operation && last_operation.state == 'in progress'
     end
 
-    def create_failed?
-      return true if last_operation&.type == 'create' && last_operation.state == 'failed'
+    def create_succeeded?
```

Comment:
You can add some tests for the new methods (create_succeeded, delete_failed, delete_in_progress) here: spec/unit/models/services/service_credential_binding_shared.rb